recipes/perl-list-moreutils-xs/meta.yaml

```
@@ -33,6 +33,10 @@ test:
   imports:
     - List::MoreUtils::XS
 
+extra:
+  additional-platforms:
+  - linux-aarch64
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)